src/atomic.rs

```diff
@@ -33,17 +36,7 @@ pub struct AtomicBitSet {
 impl AtomicBitSet {
     /// Creates an empty `AtomicBitSet`.
     pub fn new() -> AtomicBitSet {
-        AtomicBitSet {
-            layer3: AtomicUsize::new(0),
-            layer2: repeat(0)
-                .map(|_| AtomicUsize::new(0))
-                .take(1 << BITS)
-                .collect(),
-            layer1: repeat(0)
-                .map(|_| AtomicBlock::new())
-                .take(1 << (2 * BITS))
-                .collect(),
-        }
+        Default::default()
```

Comment:
Well, the question is why not. It has a default constructor (`new`), taking no parameters, so it has a default. And the main reason is that you can then easily `#[derive(Default)]` for your struct, in `specs` this is useful for `Allocator`, for example.